_contentTemplates/common/filtering.md

```diff
@@ -0,0 +1,12 @@
+#filter-row-customization-properties
+| Parameter      | Type | Description
+| ----------- | ----------- | -----------|
```

Comment:
Either use readable markdown, or minimalistic. Something in-between may look weird during future maintenance.

```suggestion
| Parameter | Type | Description |
| --- | --- | --- |
```